src/NzbDrone.Core/Datastore/Migration/210_add_monitored_seasons_filter.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using Dapper;
+using FluentMigrator;
+using Newtonsoft.Json.Linq;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(210)]
+    public class add_monitored_seasons_filter : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Execute.WithConnection(ChangeHasUnmonitoredSeason);
+        }
+
+        private void ChangeHasUnmonitoredSeason(IDbConnection conn, IDbTransaction tran)
+        {
+            var updated = new List<object>();
+            using (var getUnmonitoredSeasonFilter = conn.CreateCommand())
+            {
+                getUnmonitoredSeasonFilter.Transaction = tran;
+                getUnmonitoredSeasonFilter.CommandText = "SELECT \"Id\", \"Filters\" FROM \"CustomFilters\" WHERE EXISTS (SELECT 1 FROM json_each(\"Filters\") AS EACH WHERE json_extract(EACH.value, '$.key') = 'hasUnmonitoredSeason')";
```

Comment:
For Postgres, the syntax is different.

For SQLite, `json_extract` is not guaranteed to exist in all versions that Sonarr currently functions on because it wasn't included by default until 3.38, instead perform the JSON operations within C#.

Reply:
I just removed the extra portion and made it SELECT * FROM CustomFilters
Dont really think the limiting is needed as all filters will need to be looped over either way, and it is already being checked in the for each loop.